test/cmdlineTests/standard_ir_ast_interface_requested/output.json

```diff
@@ -0,0 +1,15 @@
+{
+    "contracts": {
+        "I": {
+            "I": {
+                "irAst": null,
+                "irOptimizedAst": null
```

Comment:
I decided to add the tests because I want to discuss their output. Currently, we are returning `null` instead of an empty object, and I’m not sure if this is the intended behavior.

Reply:
I checked and it indeed used to be `null`. So all good on that front, I find it clearer anyways.